docs/DeveloperGuide.md

```diff
@@ -35,4 +212,21 @@
 
 ## Instructions for manual testing
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+### Adding a student to the student list
+1. Adding a student with a name and phone number only
+   1. Prerequisites: View if a student with the name 'joe' exists by using the `view joe` command.
+   2. Test case: `add joe` and when prompted for phone number enter `11111111`, while pressing enter to skip other optional fields.
+   <br /> 
+   Expected: `view joe` now shows the Student details of a student with Name: joe, Phone Number: 11111111. Other fields that were left blank will reflect 'Unknown' or for date fields, today's date.
+   3. Test case: `add` and when prompted for Name, `joe`. `11111111` when prompted for phone number, press enter to skip other fields.
+   <br />
+   Expected: `view joe` shows the same results as when a student was added via `add joe`.
+### Viewing a student's details
+1. View a student who has been added to the student list
+   1. Prerequisites: Add one student named 'joe' to the list with the `add` command
+   2. Test case: `view joe`
+   <br />
+   Expected: Student's details shown correspond to the details input when `add` was used to add a student.
+   3. Test case: `view dogman`
+   <br />
+   Expected: No details are displayed, an error message stating 'Student not found!' is shown.
```

Comment:
Perhaps this section can be supported with code blocks showing the expected outputs that corresponds to the inputs given instead to make things clearer?